activities/member-relations/collect-information.md

```diff
@@ -0,0 +1,32 @@
+---
+type: Resource
+---
+
+# What information to collect about members for member relations 
+
+## Introduction
+
+This resource outlines the information that Foundation for Public Code staff use to map member relations.
+
+The information collected will be used to structure and keep track of our ongoing relationships with members. It is based on our [mission](../../organization/mission.md) and the first iterations of the [founding member proposal template](founding-membership-proposal.md). In line with our communication principles, this should be concise and hypertextual where possible. 
+
+As information collected may include personal or sensitive information, this is not public but stored in a private and secure environment.
+
+## Information collected 
+
+* Background: ```Lists existing work and/or projects relevant to the Foundation's mission.``` 
```

Comment:
While this is correct you only need one `` ` `` backtick for a code block that is inline, the triple backtick ` ``` ` is for multiline code blocks.

The additional question I have is whether it is necessary to put these in code blocks, I do not see any semantic value in it.

Spec: https://daringfireball.net/projects/markdown/syntax#code
Examples (plus GitHub's interpretation of the spec): https://github.github.com/gfm/#fenced-code-blocks

Reply:
Aha! thanks for the tips. Do you have any alternative suggestions for how to distinguish between 'hard' template structure and the 'soft' explanation bits that can be deleted?

Reply:
I wouldn't differentiate, I think it is both self-explanatory as well as not a problem if the information stays, then it can add some extra context.

Depending on the end-format you could consider adding a copy-pastable block of markdown code at the bottom for instance.

Reply:
done!